src/docs/sdk/performance/index.mdx

```diff
@@ -220,6 +222,16 @@ Besides the [usual reasons to use \*defer](https://github.com/apache/incubator-z
 
 Which in reality is useful for proxies to set it to `0` and opt out of tracing.
 
+## Header `traceparent`
+
+The header is used for trace propagation. SDKs use the header to continue traces from upstream services (e.g. incoming HTTP requests), and to propagate tracing information to downstream services (e.g. outgoing HTTP requests).
+
+`traceparent = version-traceid-spanid-traceflags`
```

Comment:
Should we explain `version` and `traceflags` a bit more? 
From our earlier conversation and skimming over the spec, `version` should be set to `00` and `traceflags` to `00`/`01` - correct?

Reply:
Let me link to https://www.w3.org/TR/trace-context/#traceparent-header here as well.